webui/react/src/components/Navigation.tsx

```diff
@@ -16,11 +16,11 @@ import css from './Navigation.module.scss';
 import NavigationSideBar from './NavigationSideBar';
 import NavigationTabbar from './NavigationTabbar';
 
-interface Props {
-  children?: React.ReactNode;
+interface Props extends PropsWithChildren {
```

Comment:
qq: why do we wanna use `PropsWithChildren`? i think `children` is preferable

Reply:
That was chosen so we could also pass `clusterMessagePresent`, which isn't a child. Is there a better way to do that?

Reply:
```tsx
interface Props {
  children?: React.ReactNode;
  isClusterMessagePresent?: boolean;
}
```
i think this should work